src/widget/wcuemenupopup.cpp

```diff
@@ -42,6 +42,12 @@ WCueMenuPopup::WCueMenuPopup(UserSettingsPointer pConfig, QWidget* parent)
     m_pDeleteCue->setObjectName("CueDeleteButton");
     connect(m_pDeleteCue, &QPushButton::clicked, this, &WCueMenuPopup::slotDeleteCue);
 
+    // FIXME(XXX): icon instead of text ?
```

Comment:
Isn't there a standard icon you could use for this? Maybe `QIcon::fromTheme("window-close")`?

Reply:
Note that this might return an invalid icon if the system's icon theme cannot be determined or lacks this icon, so we need to fall back to text in that case.

Reply:
We can alter skin the icon from the qss